src/edu/stanford/dlss/was/WasapiFile.java

```diff
@@ -4,7 +4,7 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-@SuppressWarnings("checkstyle:LineLength")
+@SuppressWarnings({"checkstyle:LineLength", "checkstyle:MethodCount", "checkstyle:MethodLength"})
```

Comment:
I believe it would be better to add the suppress warning for method length for the one method needing it, so roughly line 106 or so